src/GiftedChat.js

```diff
@@ -489,6 +489,7 @@ GiftedChat.defaultProps = {
   renderTime: null,
   user: {},
   bottomOffset: 0,
+  minInputToolbarHeight: 0,
```

Comment:
Why not put here the value of `MIN_INPUT_TOOLBAR_HEIGHT` and remove it? This also will allow get rid of `setMinInputToolbarHeight` and `this._minInputToolBarHeight`, using directly `this.props.minInputToolbarHeight` into `getMinInputToolbarHeight`.